tools/gencromfs.c

```diff
@@ -111,7 +111,7 @@
 #define NUTTX_IFLNK        (10 << 12)  /* Must match NuttX's S_IFLNK */
 
 #define DIR_MODEFLAGS      (NUTTX_IFDIR | NUTTX_IRXUSR | NUTTX_IRXGRP | NUTTX_IRXOTH)
-#define DIRLINK_MODEFLAGS  (NUTTX_IFLNK | DIR_MODEFLAGS)
+#define DIRLINK_MODEFLAGS  (NUTTX_IFLNK | NUTTX_IRXUSR | NUTTX_IRXGRP | NUTTX_IRXOTH)
```

Comment:
This part of the PR looks correct.  The rest is wrong.  Can we commit this separately?

Reply:
Done see https://github.com/apache/incubator-nuttx/pull/1551

Reply:
Okay... I created #1552.  I will close #1552